app/templates/frameworks/_agreement_returned_legal.html

```diff
@@ -3,22 +3,24 @@
     <h2 class="summary-item-heading">Legal documents</h2>
 
   {% if framework.frameworkAgreementVersion %}
-    <div class="dmspeak">
-    {% if countersigned_agreement_file %}
-      <p class="govuk-body">
-        Your original and counterpart signature pages, and the standard framework
-        agreement, form a complete, legal contract.
-      </p>
-    {% else %}
-      <p class="govuk-body">
-        Your framework agreement signature page has been sent to
-        the Crown Commercial Service (<abbr title="Crown Commercial Service">CCS</abbr>).
-        They’ll check and countersign the agreement, or contact you if there’s a
-        problem. You don’t need to wait for <abbr title="Crown Commercial Service">CCS</abbr> to
-        countersign your agreement before you start selling services.
-      </p>
-    {% endif %}
-    </div>
+      {% if signed_agreement_document_name %}
+            <div class="dmspeak">
```

Comment:
Don't think we need this div here. We're getting rid of `dmspeak` classes elsewhere in favour of standard govuk frontend spacing, so may as well remove it here if we're not relying on the visual difference.